src/consts/mpt.jl

```diff
@@ -0,0 +1,339 @@
+# Compile-time constants
+# From HPE MPT 2.23's `mpi.h` and `mpio.h` on HLRS Hawk (2022-04-22)
+
+# Implementation limits:
+const MPI_MAX_DATAREP_STRING = Cint(128)
+const MPI_MAX_ERROR_STRING = Cint(256)
+const MPI_MAX_INFO_KEY = Cint(255)
+const MPI_MAX_INFO_VAL = Cint(1024)
+const MPI_MAX_LIBRARY_VERSION_STRING = Cint(256)
+const MPI_MAX_OBJECT_NAME = Cint(128)
+const MPI_MAX_PORT_NAME = Cint(256)
+const MPI_MAX_PROCESSOR_NAME = Cint(256)
+
+# Types
+
+# Various (signed) integer types:
+const MPI_Aint   = Clong
```

Comment:
Are these the actual types used in `mpi.h`? That is, will these types be correct on both 32-bit and 64-bit architectures?

Reply:
Do you mean the four types `MPI_Aint`, `MPI_Fint`, `MPI_Count`, and `MPI_Offset`? 
Here, I use
```julia
const MPI_Aint   = Clong
const MPI_Fint   = Cint
const MPI_Count  = Clonglong
const MPI_Offset = Clonglong
```
while in MPT's `mpi.h` I found the following (pulled together from different parts of the file):
```c
typedef long      MPI_Aint;
typedef int       MPI_Fint;
typedef long long MPI_Count;
typedef long long MPI_Offset;
```
As a sanity check, I also compiled `` from and got this
```julia
const MPI_Aint = Int64
const MPI_Count = Int64
const MPI_Fint = Int32
const MPI_Offset = Int64
```
Thus at least for "regular" 64-bit Linux systems the `Cxxx` types seem to be correct. Or am I missing something?

Reply:
I wanted to double-check that the Julia declarations are consistent with the C declarations. They are – thanks!